Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG APP_INSIGHTS_AGENT_VERSION=3.2.10
 FROM hmctspublic.azurecr.io/base/java:17-distroless
 
-COPY build/libs/spring-boot-template.jar /opt/app/
+COPY build/libs/pip-cron-trigger.jar /opt/app/
 
-EXPOSE 4550
-CMD [ "spring-boot-template.jar" ]
+EXPOSE 8050
```

Comment:
.env file will need updating to be this port :)

Reply:
Updated to new port